Clicker/Views/Cards/PollOptionsCell.swift

```diff
@@ -50,14 +57,23 @@ class PollOptionsCell: UICollectionViewCell {
         adapter = ListAdapter(updater: updater, viewController: nil)
         adapter.collectionView = collectionView
         adapter.dataSource = self
+        adapter.scrollViewDelegate = self
+    }
+    
+    func scrollViewDidScroll(_ scrollView: UIScrollView) {
+        let diff = collectionView.contentSize.height - bounds.height - scrollView.contentOffset.y
+        if diff < 10 {
+            
```

Comment:
What's going on here? This looks not finished...

Reply:
Not done with that yet. I'll get it done before the meeting tomorrow.